docs/api-reference/chat.md

```
@@ -0,0 +1,274 @@
+
+### Request Body
+
+- **Required:** No
+
+- **Content-Type:** `application/json`
+  | Name | Type | Required | Description |
+  |------|------|----------|-------------|
+  | `inputs` | object | ❌ No | - |
+  | `data` | object | ❌ No | - |
+  | `files` | array[string] | ❌ No | - |
+  **`inputs`** ❌
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)